core/cloudflow-cli/src/main/scala/akka/cli/cloudflow/Cli.scala

```diff
@@ -16,9 +16,7 @@ object Cli {
   def defaultKubeClient(config: Option[File], logger: CliLogger) =
     new KubeClientFabric8(config)(logger)
 
-  val ProtocolVersion = "6"
-
-  val SupportedApplicationDescriptorVersion = 6
+  val ApplicationDescriptorVersion = 6
```

Comment:
7?

Reply:
even `ApplicationDescriptor` ?

Reply:
Oh wait. No you are right, this stays the same.